src/Notecard.cpp

```diff
@@ -309,18 +309,24 @@ void Notecard::deleteResponse(J *rsp)
 
 /**************************************************************************/
 /*!
+    @deprecated Do not use.
```

Comment:
> "Do not use."

should probably be replaced with:

> "NoteDebug should be treated as an internal Notecard logging function, used only by the library itself (note-arduino and note-c) and not its users."

https://www.doxygen.nl/manual/commands.html#cmddeprecated